README.md

```diff
@@ -126,9 +126,9 @@ below) has changed.
     debug: PropTypes.bool,
 
     /**
-     * The `throttleHandler` prop provides a way to throttle scroll callbacks
-     * to increase performance. See the section on "Throttling" for details on
-     * how to use it.
+     * The `throttleHandler` prop provides a callback that throttle internal
```

Comment:
technically is not a 'callback'. Maybe a `wrapped function` or `composed function`?
I don´t know for sure.